client/post-editor/media-modal/index.jsx

```diff
@@ -210,7 +210,7 @@ module.exports = React.createClass( {
 		this.setView( ModalViews.IMAGE_EDITOR );
 	},
 
-	onImageEditorDone( error, blob, imageEditorProps ) {
+	onImageEditorDone( error, blob, imageEditorProps, media ) {
```

Comment:
Also, IDs are a general concept and I think every `media` passed to image editor, whether in media modal or anywhere else should have some ID.